x-pack/test/integrations_manager_api_integration/apis/image.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import ServerMock from 'mock-http-server';
+import { FtrProviderContext } from '../../api_integration/ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  describe('images', () => {
+    const server = new ServerMock({ host: 'localhost', port: 6666 });
+    beforeEach(() => {
+      server.start(() => {});
+    });
+    afterEach(() => {
+      server.stop(() => {});
+    });
+    it('fetches a png screenshot image from the registry', async () => {
+      server.on({
+        method: 'GET',
+        path: '/package/auditd-2.0.4/img/screenshots/auditbeat-file-integrity-dashboard.png',
+        reply: {
+          headers: { 'content-type': 'image/png' },
+        },
+      });
+
+      const supertest = getService('supertest');
+      const fetchImage = async () => {
```

Comment:
Could also remove this `fetchImage` wrapping function and `await supertest` directly but that's a style, not substance, comment